src/net/sourceforge/kolmafia/RequestLogger.java

```diff
@@ -21,7 +22,6 @@ public class RequestLogger extends NullStream {
   public static final RequestLogger INSTANCE = new RequestLogger();
 
   private static PrintStream outputStream = KoLmafiaTUI.outputStream;
-  private static PrintStream mirrorStream = NullStream.INSTANCE;
```

Comment:
This is not what I was expecting.  If the problem is something written to the gCLI that is not mirrored and should be then I would expect a line like RequestLogger.mirrorStream.println(message); just needs to be every place something is written to the gCLI.  A discussion of your approach might help me.

Reply:
Reverted the changes to using ChatBuffer